.github/maintainers_guide.md

```diff
@@ -111,63 +111,101 @@ If you want to test the package locally you can.
 It is possible to deploy this project on <https://test.pypi.org/> in order to try out packaging and deploy related changes
 
 ```zsh
-./scripts/deploy_to_test_pypi.sh
+# Set the new version with SLACK_CLI_HOOKS_VERSION
+SLACK_CLI_HOOKS_VERSION="1.2.3" ./scripts/deploy_to_test_pypi.sh
 ```
 
 #### Development Deployment
 
-1. Create a branch in which the development release will live:
-    - Bump the version number in adherence to [Semantic Versioning](http://semver.org/) and [Developmental Release](https://peps.python.org/pep-0440/#developmental-releases) in `slack_cli_hooks/version.py`
-      - Example the current version is `1.2.3` a proper development bump would be `1.3.0.dev0`
-      - `.dev` will indicate to pip that this is a [Development Release](https://peps.python.org/pep-0440/#developmental-releases)
-      - Note that the `dev` version can be bumped in development releases: `1.3.0.dev0` -> `1.3.0.dev1`
-    - Commit with a message including the new version number. For example `1.3.0.dev0` & Push the commit to a branch where the development release will live (create it if it does not exist)
-      - `git checkout -b future-release`
-      - `git add slack_cli_hooks/version.py`
-      - `git commit -m 'version 1.3.0.dev0'`
-      - `git push future-release`
-    - Create a git tag for the release. For example `git tag 1.3.0.dev0`.
-    - Push the tag up to github with `git push origin --tags`
-
-2. Distribute the release
-   - Use the latest stable Python runtime
-   - `python -m venv .venv`
-   - `scripts/deploy_to_prod_pypi.sh`
-   - You do not need to create a GitHub release
-
-3. (Slack Internal) Communicate the release internally
+Releases for this library are automatically generated off of git releases. Before
+creating a new release, ensure that everything on a feature branch,
+[run the tests](#run-all-the-unit-tests).
```

Comment:
The wording is a little confusing here- not super sure what should be on the feature branch 😜

Reply:
Good point I'll change this to `stable branch`?

Reply:
Either `stable` or `feature` would be good IMO. I was poking more at if we want to make sure "everything on a x branch **has landed**, then run the tests" or something else?